pkg/virtctl/vm/add_volume_test.go

```diff
@@ -185,3 +166,27 @@ var _ = Describe("Add volume command", func() {
 		Entry("addvolume dv, with LUN-type disk should call VMI endpoint", "addvolume", "testvmi", "testvolume", true, expectVMIEndpointAddVolume, "--disk-type", "lun"),
 	)
 })
+
+func createTestDataVolume() *v1beta1.DataVolume {
```

Comment:
Can you drop this func and inline `libdv.NewDataVolume(libdv.WithName(volumeName))` into L141? I don't think we should carry this func for just a single caller.

Reply:
I don't think `createTestDataVolume` makes a huge difference compared to `...NewDataVolume..WithName...`. Why hide the truth and add unnecessary lines of code?

Reply:
hiding reduce mental burden when reading the code
this is the exact purpose of functions, allow the reader the option to focus on what he would like
anyhow i will change